olive/engine/engine.py

```diff
@@ -758,15 +764,19 @@ def _run_pass(
         # new model id
         input_model_number = input_model_id.split("_")[0]
         output_model_id = f"{self._get_new_model_number()}_{pass_name}-{input_model_number}-{hash_dict(pass_config)}"
-        output_model_path = str(self._model_cache_path / f"{output_model_id}")
+        output_model_path = self._model_cache_path / f"{output_model_id}" / "output_model"
```

Comment:
the output_model_id need to be modified to add the accelerator suffix if the pass is not accelerator agnostic. 
https://github.com/microsoft/Olive/pull/286. 

How do we handle this scenario if the output model name changes from model id to hardcode output_model?

Reply:
How could my PR for reruse run logic looks like by including your PR?

Reply:
There is no effect as we discussed offline. Resolving this conversation.